package.json

```diff
@@ -83,7 +83,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10"
+    "node": ">=12.9.0"
```

Comment:
This choice is based on:

Node 12.9 updated its v8 version: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#2019-08-20-version-1290-current-targos

And these are the changes that it brings about: https://v8.dev/blog/v8-release-76